opendrift/models/basemodel.py

```diff
@@ -3732,22 +3732,52 @@ def plot_timestep(i):
             blit = False  # Blitting does not work on mac
         else:
             blit = False  # Must return artists before this is activated
-        anim = animation.FuncAnimation(plt.gcf(),
-                                       plot_timestep,
-                                       blit=blit,
-                                       frames=x.shape[0],
-                                       interval=50)
-
-        if filename is not None or 'sphinx_gallery' in sys.modules:
-            self._save_animation(anim, filename, fps, fastwriter=fastwriter)
-            logger.debug('Time to make animation: %s' %
-                         (datetime.now() - start_time))
-        else:
+
+        frames=x.shape[0]
+        if compare is not None:
+            frames=min(x.shape[0],cd['x_other'].shape[1])
+
+        mp4=False
+        if filename is not None: mp4 = str(filename)[-4:]=='.mp4'
```

Comment:
Når du skal sjekke extension so er det betre å bruke `pathlib.Path.suffix`: https://docs.python.org/3/library/pathlib.html#pathlib.PurePath.suffix. Slik det er no vil det krasje dersom filnavn har færre enn 4 chars.

Reply:
![image](https://user-images.githubusercontent.com/65763988/144230931-786e7316-9be7-42b3-ba43-7e6e20ee087e.png)

Reply:
Du har rett, uansett so bør vi bruke python sin funksjonalitet for å sjekke extensions :)